ledis/t_list.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/siddontang/go/log"
 	"github.com/siddontang/go/num"
 	"github.com/siddontang/ledisdb/store"
+	"golang.org/x/net/context"
```

Comment:
I prefer using go context directly.

Btw, we can upgrade go verion in traivs CI too.